docs/technical-documentation/migration-overview.md

```diff
@@ -0,0 +1,151 @@
+## Islandora 8 Migration Overview
+
+This video provides an overview of the various options available to migrate data into an Islandora 8 installation.
+
+[![Islandora Online: Islandora Migration Tools](https://img.youtube.com/vi/95Bnix-z1zY/0.jpg)](https://www.youtube.com/watch?v=95Bnix-z1zY)
+
+The three options are:
+- REST API
+- Migrate API
+- Islandora Workbench
+
+### REST API
+
+Why use the rest API?
+- Works anywhere
+  - You don’t have to work on the Drupal server. Migrate from your laptop!
+- No PHP required
+  - Use any language that can make an http request. Even cURL will do just fine.
+- JSON
+  - Why use XML if you don’t have to?
+
+BONUS: It’s just Drupal’s REST API
+
+Islandora only provides two additional API endpoints:
+
+- /media/{mid}/source
+  - PUT a file to this endpoint to create/update a Media’s file
+- /node/{nid}/media/{media_type}/{taxonomy_term}
+  - PUT a file to this endpoint to create/update a Media for a Node
+
+Just be aware, you are writing everything yourself! (In other words you are making all of the migration decisions yourself.)
+
+### Migrate API
+
+Uses the Drupal 8 [Migrate API](https://www.drupal.org/docs/8/api/migrate-api/migrate-api-overview), which "provides services for migrating data from a source system to Drupal 8.
+
+The "source system" can be almost anything:
+- an Islandora 7 system
+- a group of scanned images and their metadata inside a CSV file
+- a web API
+
+Why use the Migrate API?
+
+- You can (potentially) do everything with configs!
+- Leverage contrib module plugins.
+- Making plugins for more complex sources and  processes is (relatively) simple.
+- Updating metadata is as simple as:
+`drush mim node --update`
+
+A Migration Configuration defines an Extract, Transform, Load (ETL) process.
+
+- Source plugins extract data from a source
+- Process plugins transform the data
+- Destination plugins load the data (create new entities)
+
+We’ve built two tools for you using the Migrate API:
```

Comment:
I found this section a bit hard to follow (there's a lot of information, which is awesome!). Can you add some sub-headings to break it out visually? I think if "A Migration Configuration defines an Extract, Transform, Load (ETL) process." was ####, it would read easier.

Reply:
I added more Markdown headings marks to the docs. Let me know what you think.